flavors/template-Exasol-all-python-3.10/flavor_base/derived_from

```diff
@@ -0,0 +1 @@
+flavors/python-3.8-minimal-EXASOL-6.2.0
```

Comment:
Do we need to bump the Exasol version at some point?

Reply:
No this file is for indicating from which flavor we derived this one. We need this for the package diffs.